i18n/pl.json

```diff
@@ -9,24 +10,24 @@
 		"strona": "page",
 		"szukaj": "search",
 		"🎲": "random",
-		"discussions": "discussion",
```

Comment:
That would remove the `discussions` alias for the `discussion` command. It should probably stay to keep English commands working in all languages. You can add a new key for other Polish alias instead.

Reply:
i feel bullied by two people at this point